src/components/AccountDrawer/UniwalletModal.tsx

```diff
@@ -1,14 +1,14 @@
 import { Trans } from '@lingui/macro'
 import { sendAnalyticsEvent } from '@uniswap/analytics'
 import { InterfaceElementName } from '@uniswap/analytics-events'
-import { WalletConnect } from '@web3-react/walletconnect'
+import { WalletConnect } from '@web3-react/walletconnect-v2'
```

Comment:
Does this mean that we're migrating Uniwallet connections to V2 as part of this PR?

Reply:
yes